tola/settings/local.py

```diff
@@ -70,7 +70,7 @@
 
 
 ####### Tola Activity API #######
-TOLA_ACTIVITY_API_URL = os.getenv('TOLA_ACTIVITY_API_URL')
+TOLA_ACTIVITY_API_URL = os.getenv('TOLA_ACTIVITY_API_URL', '')
 TOLA_ACTIVITY_API_TOKEN = os.getenv('TOLA_ACTIVITY_API_TOKEN')
```

Comment:
The `local.py` imports the `base.py` and these two lines are overwriting the values from `base.py`. You can add this default empty string in that file and remove these lines.